src/mixins/nft.js

```diff
@@ -100,8 +107,11 @@ export default {
     NFTClassMetadata() {
       return this.getNFTClassMetadataById(this.classId) || {};
     },
-    isWritingNFT() {
-      return isWritingNFT(this.NFTClassMetadata);
+    nftClassCollectionType() {
+      return getNFTClassCollectionType(this.NFTClassMetadata);
+    },
```

Comment:
Seems some conditional checks in `if (this.isWritingNFT)` should be refactored to `if (this.nftClassCollectionType === nftClassCollectionType.WritingNFT)`?
like [here](https://github.com/likecoin/liker-land/blob/decbf2c249742fa0f073b42c28f77d368a6ddb5c/src/mixins/nft.js#L392)